app/views/layouts/application.html.erb

```diff
@@ -58,19 +58,28 @@
             <%= nav_link FlagSettingsController, action: :dashboard, label: 'flagging', active: [FlagConditionsController, FlagLogController] %>
             <% if user_signed_in? %>
               <%= nav_dropdown AdminController, label: 'tools' do %>
+                <%= nav_link AdminController, label: 'User Data', action: :users %>
                 <%= nav_link AdminController, label: 'Invalidated Feedback', action: :recently_invalidated %>
                 <%= nav_link AdminController, label: 'Feedback via API', action: :api_feedback %>
-                <%= nav_link AdminController, label: 'User Data', action: :users %>
                 <% if current_user.moderator? %>
                   <%= nav_link StackExchangeUsersController, label: 'Spammers', action: :sites %>
                 <% end %>
+                <li role="separator" class="divider"></li>
+                <% if Announcement.current.count > 0 %>
+                  <%= nav_link AnnouncementsController, label: 'Announcements <span class="badge">' + Announcement.current.count + '</span>' %>
```

Comment:
Valid, but you can use string interpolation: `"Announcements <span class='badge'>#{Announcement.current.count}</span>"`